packages/components/basic-table/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@tradeshift/elements.basic-table",
```

Comment:
@aleks-elkin Do you think we need to unpublish the `ts-table`?

Reply:
Based on the official documentation for [unpublish](https://docs.npmjs.com/cli/unpublish.html): 
> unpublish is only allowed with versions published in the last 72 hours

So we are a bit late for unpublish :)

Also, as I understand from [this](https://stackoverflow.com/a/38632439) SO answer and `npm unpublish` documentation, the recommended way would be to deprecate the `ts-table` package and publish a new package `basic-table`.

And I suggest to take this way with deprecation and publish a new package.

Just to be clear, it will require nothing from @radu-marius-nedelcu , it's our responsibility to manage the package registry, so if you, @esmailpour-hosein , have any questions/concerns/alternatives, we can discuss it on separate meeting of our team.